vars/getCloudProviderFromBackend.groovy

```diff
@@ -6,6 +6,8 @@ def call(String backend) {
         'k8s-local-kind-aws': 'aws',
         'k8s-local-kind-gce': 'gce',
         'k8s-gce-minikube': 'gce',
+        'siren-aws': 'aws',
+        'siren-gce': 'gce',
```

Comment:
@vponomaryov can you please advice? perhaps i'm missing something here, i just added " 'siren-gce': 'gce' " but it didn't work so i had to use the above " cloud_provider = cloud_provider.replace('-siren', '') "

Reply:
You have typos here. Words in backend names are placed incorrectly. It must be "aws-siren" and "gce-siren".